common/model/opening-hours.js

```diff
@@ -22,20 +22,20 @@ export type ExceptionalPeriod = {|
 
 export type OpeningHoursDay = {|
   dayOfWeek: Day,
-  opens?: string,
-  closes?: string,
+  opens?: ?string,
```

Comment:
Always feels a bit weird to have a nullable + optional property?
Is it not one or the other? e.g. nullable, but you have to send `null` if it is?